features/skipped_steps.feature

```diff
@@ -78,3 +78,50 @@ Feature: Skipped steps
     When I run cucumber-js
     Then it passes
     And the step "a skipped step" has status "skipped"
+
+  Scenario: Skipped before hook should skip all before hooks
```

Comment:
Based on this description, it appears we should be testing that the other before skip is skipped